### ISSUE TITLE
Add .noembed to disable embeds for a guild

### ISSUE THREAD

**Issue by emmiegit** [strinking/mawabot#35]
Some guilds block embeds on their bot channels, so having a `.noembed` command could toggle the use of embeds in selfbot commands. When disabled, command outputs are done as a normal post instead.

**Comment by emmiegit** [comment]
This will also require some kind of local data storage.

**Comment by ghost** [comment]
you could just check if the server will allow you to use embeds at startup/before msg send and then set a flag for that server in memory and use plaintext from then on

**Comment by emmiegit** [comment]
That is true, some guilds allow user embeds but prohibit their usage, usually under anti-selfbot rules. My intention was to have a sqlite database or something where you can mark certain guilds as "no-embed".
